nagios/tests/test_e2e.py

```diff
@@ -0,0 +1,13 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import logging
+
+import pytest
+
+LOGGER = logging.getLogger(__name__)
+
+
+@pytest.mark.e2e
+def test_e2e(dd_agent_check, instance):
+    dd_agent_check(instance, rate=True)
```

Comment:
What does this test do ? no assertion ? 🤔

Reply:
Right, I started this to get an environment running and then forgot about the test

Reply:
The test would still fail if check didnt work :p

Reply:
Yes, so this is checking pretty much just that the agent runs successfully and that if finds and can read all the relevant files. Check does not emmit any metrics normally because the check is based on tailing files so it needs to happen right in between new lines added to the files.